libs/designsystem/src/lib/components/page-local-navigation/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './page-local-navigation.component';
+export * from './page-local-navigation-item';
```

Comment:
Always prefer explicit export, because implicit export is one of the most prominent factors for circular dependency error